packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -79,6 +81,9 @@
 VALUE_NOT_PROVIDED = object()
 # tolerance in seconds for new blocks not having arrived yet
 BLOCKS_STALL_TOLERANCE = 60
+SERIOUS_OFFENCE_ENUM_MIN = 1000
+NUMBER_OF_BLOCKS_TRACKED = 10_000
```

Comment:
Why the underscore?

Reply:
https://peps.python.org/pep-0515/   :smiley: